Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -3369,7 +3373,6 @@
 				77FD8B45216D6245000A95AC /* LoadingBarButtonItemView.swift */,
 				D63BBD382180BE5D007E01F0 /* PaymentMethodsFooterView.swift */,
 				374F507822614A1000DE6746 /* PledgeFooterView.swift */,
-				5993DEBD1CE296F000925494 /* ProfileHeaderView.swift */,
```

Comment:
These file removals must have been dangling in the project for a while...we've removed them some time back.